src/solidity-frontend/solidity_grammar.h

```diff
@@ -50,6 +50,37 @@ enum ElementaryTypeNameT
 {
   // rule unsigned-integer-type
   UINT8 = 0,
+  UINT16 = 1,
```

Comment:
`= 1,` is not needed here. Same for the other items in this enum.

Reply:
Will amend